MyMentorProject/src/main/java/com/Deny/MyMentorProject/User/User.java

```diff
@@ -0,0 +1,49 @@
+package com.Deny.MyMentorProject.User;
+
+
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Remove this as User is constantly changing. Only classes which are stateless should be initialized as a Spring Bean.